man/ostree-config.xml

```diff
@@ -62,20 +62,42 @@ Boston, MA 02111-1307, USA.
             <cmdsynopsis>
                 <command>ostree config set</command> <arg choice="req"> --group=GROUPNAME</arg> <arg choice="req"> KEYNAME</arg> <arg choice="req">VALUE</arg>
             </cmdsynopsis>
+            <cmdsynopsis>
+                <command>ostree config unset</command> <arg choice="req">SECTIONNAME.KEYNAME</arg>
+            </cmdsynopsis>
+            <cmdsynopsis>
+                <command>ostree config unset</command> <arg choice="req"> --group=GROUPNAME</arg> <arg choice="req"> KEYNAME</arg>
```

Comment:
Aside: I'm confused now why we use both the `GROUPNAME` and `SECTIONNAME` nouns when they're the same thing, right? I'd vote for harmonizing those to make it the same. Anyway, just writing this down so I don't forget, doesn't have to be in this patch!

Reply:
Added a commit to fix this